.github/pull_request_template.md

```diff
@@ -2,12 +2,20 @@
 
 > Why this pull request was created? Explain the problem that this pull request is trying to solve.
 
+[comment]: # (Add explanation bellow this line)
```

Comment:
Is there any reason why `<!-- comment -->` syntax is not used?

It is even used even by `markdownlint` for configure file linting: https://github.com/DavidAnson/markdownlint#configuration

Reply:
I found this syntax on StackOverflow - [Comments in Markdown](https://stackoverflow.com/a/32190021/4944847).

If the `<!-- comment -->` syntax is better, I can use it 🙂

Reply:
Personally, I would go with `<!-- HTML comment  syntax -->` as it is less noisy and more flexible.

It is also covered by https://spec.commonmark.org/0.29/

Moreover, markdownlint assumes that this syntax is supported.

Reply:
I updated PR to use HTML comment syntax